cmd/reconciler-manager/main.go

```diff
@@ -113,7 +113,7 @@ func main() {
 	}
 
 	// Register the OC Agent exporter
-	oce, err := metrics.RegisterOCAgentExporter()
+	oce, err := metrics.RegisterOCAgentExporter(reconcilermanager.ManagerName)
 	if err != nil {
 		setupLog.Error(err, "failed to register the OC Agent exporter")
```

Comment:
Shold this use klog.Fatalf like the others? What's the setuplog for?

Reply:
As discussed - going to unify the klog in a separate PR